packages/server/src/api/controllers/application.ts

```diff
@@ -197,6 +199,30 @@ export async function fetch(ctx: UserCtx) {
     }
   }
 
+  // Get all builder sessions in each app
+  const sessions = await builderSocket?.getRoomSessions(appIds)
+  if (sessions?.length) {
+    let appSessionMap: { [key: string]: SocketSession[] } = {}
```

Comment:
Typing for this which is a bit nicer: `Record<string, SocketSession[]>`

Reply:
Nice, TIL that this is just a nicer shorthand way of doing that.